src/ui-shell/ui-shell.stories.ts

```diff
@@ -303,6 +309,51 @@ const SideNavigationRouterTemplate = (args) => ({
 export const SideNavigationRouter = SideNavigationRouterTemplate.bind({});
 SideNavigationRouter.storyName = "Side Navigation with router";
 
+
+const SideNavigationUseRouterTemplate = (args) => ({
+	props: args,
+	styles: [`
+		::ng-deep .cds--side-nav__item--active {
+			background: var(--cds-background-selected);
+
+			&:before {
+			    position: absolute;
+    			background-color: var(--cds-border-interactive, #0f62fe);
+    			content: "";
+    			inline-size: 3px;
+    			inset-block-end: 0;
+    			inset-block-start: 0;
+    			inset-inline-start: 0;
+    		}
+		}
+	`],
```

Comment:
I don't think we need this. After looking at @carbon/styles, seems like the `active` class is assigned when `aria-current="page"`. Could you please verify this when router is enabled?

Reply:
You are also correct, this was probably an issue before I correctly added the `aria-current` thanks for noticing!